esdl/providers/sentinel1.py

```diff
@@ -10,51 +9,57 @@
 
 
 class S1Provider(NetCDFCubeSourceProvider):
-    def __init__(self, cube_config, name='sentinel1', dir=None, resampling_order=None, polarisation=None):
-        super(S1Provider, self).__init__(cube_config, name, dir, resampling_order)
+
+    def __init__(self, cube_config, name='sentinel1', dir=None, resampling_order=None, polarisation=None, orbit=None):
+        super(S1Provider, self).__init__(
+            cube_config, name, dir, resampling_order)
         self.old_indices = None
         self.pol = polarisation
+        if orbit not in ['A', 'D']:
+            raise ValueError(
```

Comment:
We should also allow the option to ignore the orbit. Maybe this could be just "".
Also it is confusing, that you are checking for uppercase input and then convert this to lowercase. I think this should also allow lowercase input.